src/box/lua/call.c

```diff
@@ -1268,6 +1269,56 @@ lbox_func_new_or_delete(struct trigger *trigger, void *event)
 	return 0;
 }
 
+static int
+lbox_grant_rt_access(struct lua_State *L)
+{
+	if (lua_gettop(L) != 2)
+		return luaL_error(L, "internal.grant_lua_call bad arguments. "
+				  "Expected exactly two: user name and "
+				  "function name.");
+
+	struct credentials *grantor = effective_user();
+	struct user *grantor_user = user_find(grantor->uid);
+	if (grantor->uid != ADMIN && is_super(grantor_user) == 0)
+		return luaL_error(L, "internal.grant_lua_call only ADMIN or "
+				  "user with role SUPER can grant access.");
```

Comment:
This check is inconsistent with permissions to change permissions in the `_priv` system space.

https://github.com/tarantool/tarantool/blob/8947cb04f59423e2944d48b8a1effec2fb11b1db/src/box/alter.cc#L3955-L3961

I don't really know, whether it is good or bad to forbid such actions for the `super` role, but I'm for consistency.

Reply:
now only `admin` can grant/revoke access.